official/transformer/transformer_main.py

```diff
@@ -46,14 +46,18 @@
 from official.utils.flags import core as flags_core
 from official.utils.logs import hooks_helper
 from official.utils.logs import logger
+from official.utils.misc import distribution_utils
 from official.utils.misc import model_helpers
 
-
 PARAMS_MAP = {
     "tiny": model_params.TINY_PARAMS,
     "base": model_params.BASE_PARAMS,
+    "base_multi_gpu": model_params.BASE_MULTI_GPU_PARAMS,
     "big": model_params.BIG_PARAMS,
+    "big_multi_gpu": model_params.BIG_MULTI_GPU_PARAMS,
```

Comment:
I feel like it might be better to exclude these from choices, and hot-swap them out if there are multiple GPUs, logging a message so that people know it happened. Thoughts?

Reply:
Yeah, I was conflicted about the choice between swapping the parameters vs creating a separate set of params were about equal. Usability-wise, I prefer swapping the params in the multi-gpu case so the running commands don't have to change as much. I ended up choosing the other option to maintain the idea to keep models "mathematically equivalent" when swapping to multiple GPUs (like how the batch size is global instead of per-device). 

I don't have strong preferences. Do you still think we should go with swapping the params?

Reply:
I was thinking have both sets of params, but don't include the multi-GPU in the mapping dict. Then, if multi GPU, swap the entire param set up front.